app/views/admin/people/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= simple_form_for :people, url: batch_update_admin_people_path, method: :put do |f| %>
+  <table>
+    <tr>
+      <th>Name</th>
+      <th>Admin</th>
+    </tr>
+    <% @people.each do |person| %>
+      <tr>
+        <td><%= person.first_name %></td>
+        <td>
+          <%= simple_fields_for person do |f| %>
+            <%= f.input :id, as: :hidden, input_html:
+              { name: "people[person-#{person.id}]id" } %>
+            <%= f.input :admin?, as: :boolean, label: false, input_html:
```

Comment:
Not sure `admin?` is a valid field name..... what does this look like when its gets sent as params